AnnoyancesFilter/sections/antiadblock.txt

```diff
@@ -534,6 +534,7 @@ tv100.com,thehimalayantimes.com,aleph.org.mx,realmtracker.com,iteramos.com,telly
 !
 ! generic unblocks
 !#safari_cb_affinity(general)
+@@||brainly.com/app/scripts/wp-banners.js
```

Comment:
What about `brainly.com##.brn-blocker-box`? Please check it.

Reply:
No. I think the reporter wants to remove anti-adblock script when you follow the following instruction:
1. Visit the provided URL.
2. Click the highlighted button.
<details>
<summary>Screenshot</summary>

![2932](https://user-images.githubusercontent.com/37463155/141704909-6e8c86f8-cf3f-4b6f-bd19-bf16bc994277.png)

</details>

Reply:
Moved the rule to the content blockers sections because I'd added scriptlet.